web3/manager.py

```diff
@@ -87,22 +99,24 @@ def _make_request(self, method, params):
         self.logger.debug("Making request. Method: %s", method)
         return await request_func(method, params)
 
-    def request_blocking(self, method, params):
+    def request_blocking(self, method, params, error_formatters=None):
         """
         Make a synchronous request using the provider
         """
         response = self._make_request(method, params)
+        apply_error_formatters(error_formatters, response)
 
         if "error" in response:
             raise ValueError(response["error"])
 
         return response['result']
 
-    async def coro_request(self, method, params):
+    async def coro_request(self, method, params, error_formatters=None):
         """
         Couroutine for making a request using the provider
         """
         response = await self._coro_make_request(method, params)
+        apply_error_formatters(error_formatters, response)
```

Comment:
Interesting.  I would have expected this to be inside of the `if "error" in response` block below.  Is there a reason it is here?

Reply:
Hm, good catch. Not that I can tell. Moved!